x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -833,6 +834,22 @@ export class TimeSeriesExplorer extends React.Component {
   }
 
   componentDidMount() {
+    // if timeRange used in the url is incorrect
+    // perhaps due to user's advanced setting using incorrect date-maths
+    const { invalidTimeRangeError } = this.props;
+    if (invalidTimeRangeError) {
+      const toastNotifications = getToastNotifications();
+      toastNotifications.addWarning(
+        i18n.translate('xpack.ml.timeSeriesExplorer.invalidTimeRangeInUrlCallout', {
+          defaultMessage:
+            'The time filter changed to the full range for this job due to invalid default time filter. Please check the advanced settings for {field}.',
```

Comment:
Nit, as above suggest changing to `The time filter was changed to the full range for this job due to an invalid default time filter. Please check the advanced settings for {field}.` No need to check for plural case as there will only ever be 1 job selected in this view.

Reply:
Updated here https://github.com/elastic/kibana/pull/76347/commits/1118b3e08345110bec102f86bfbd3bf46cce969c